connor/not.go

```diff
@@ -0,0 +1,13 @@
+package connor
+
+// not is an operator which performs object equality test
+// and returns the inverse of the result.
+func not(condition, data any) (bool, error) {
+	if m, err := eq(condition, data); err != nil {
+		return false, err
+	} else if m {
+		return false, nil
+	}
+
+	return true, nil
+}
```

Comment:
nitpick: Could just be:

```suggestion
func not(condition, data any) (bool, error) {
	if m, err := eq(condition, data); err != nil {
		return false, err
	}
	return !m, nil
}
```

Reply:
I like this. Thank you for the suggestion.

Reply:
✅